ansible/inventory.yml

```diff
@@ -183,8 +187,7 @@ hosts:
           ubuntu1604-x64-1: {ip: 51.15.76.107}
 
       - softlayer:
-          ubuntu1604-x64-1: {ip: 169.55.150.155}
           rhel74-x64-1: {ip: 169.55.170.70}
           rhel69-x64-1: {ip: 169.55.150.147}
-          win2012r2-x64-1: {ip: 169.55.170.72, user: admin}
+          ubuntu1604-x64-1: {ip: 169.55.150.155}
```

Comment:
Windows machine `169.55.170.72` is still live in softlayer and allocated to us so I don't think these should be removed

Reply:
I'll reinstate it :)